### ISSUE TITLE
Unchecked checkboxes and radio buttons have low contrast

### ISSUE THREAD

**Issue by Paul-Hebert** [cloudfour/cloudfour.com-patterns#461]
These form elements may be difficult to use due to their low contrast. (They get a 1.25 contrast ratio on a white background)

We should investigate using a darker color or increasing contrast some other way to make these easier to see and use (This will help accessibility for sight impaired users as well as users in high glare situations)

<img width="451" alt="Screen Shot 2019-10-16 at 2 12 46 PM" src="https://user-images.githubusercontent.com/5798536/66959622-9b336580-f01f-11e9-8ade-2355cd58d36d.png">

**Comment by tylersticka** [comment]
This may not be _quite_ as straightforward as I thought it would be.

Here's what one of these would look like in the context of a current form if we darken the border colors to the nearest gray value that passes WCAG AA:

<img width="788" alt="Screen Shot 2019-10-16 at 2 59 09 PM" src="https://user-images.githubusercontent.com/69633/66962415-c5882180-f025-11e9-8f3d-01831dc793ec.png">

It's definitely easier to spot, but now it feels a bit inconsistent with the preceding inputs.

We could darken those, too, but this exposes an issue we have where the color is the main thing distinguishing focused fields:

<img width="788" alt="Screen Shot 2019-10-16 at 3 02 03 PM" src="https://user-images.githubusercontent.com/69633/66962482-f5cfc000-f025-11e9-8bce-e3702eee13e9.png">

This made me take a step back and ask myself whether or not holding these to the same contrast standards as _text_ is even necessary?

For example, here's the _default_ unchecked checkbox in MacOS/Firefox:

<img width="395" alt="Screen Shot 2019-10-16 at 3 04 42 PM" src="https://user-images.githubusercontent.com/69633/66962614-4cd59500-f026-11e9-8b9f-88ebcaa24aa4.png">

It's a fraction of the size, and it _also_ fails to meet WCAG contrast suggestions.

This led me to research contrast recommendations for non-text elements.

I found [this W3C page](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Contrast_(Minimum)) that mentions there are lower contrast recommendations for elements with a border of `3px` or larger (which would include our toggles). This seems to gel with [the "Non-Text Contrast" section of the WCAG guidelines](https://www.w3.org/TR/WCAG21/#non-text-contrast).

If we follow those recommendations, I think this new, darker gray (3.06:1 contrast) might be a nice compromise:

<img width="782" alt="Screen Shot 2019-10-16 at 3 12 34 PM" src="https://user-images.githubusercontent.com/69633/66963020-7511c380-f027-11e9-921d-7dacf63f5db1.png">

(Thoughts and feedback welcome!)

**Comment by Paul-Hebert** [comment]
Thanks for looking into this @tylersticka !

The darker grey looks like a good compromise to me but I'll defer to those who have more experience with the C4 brand.
